lib/tasks/dev.rake

```diff
@@ -0,0 +1,63 @@
+require 'rspec/core'
+require 'rspec/core/rake_task'
+require 'solr_wrapper'
+require 'fcrepo_wrapper'
+require 'active_fedora/rake_support'
+
+module DevRakeUtil
+  def load_config(service, environment, defaults)
```

Comment:
oops, this is unnecceary and didn't need to be committed. Was a false path. Thanks!